test/e2e/cross_os/test_job_submissions.py

```diff
@@ -457,19 +468,63 @@ def test_worker_uses_job_attachment_configuration(
             {"name": "StringToAppend", "value": test_run_uuid},
             {"name": "DataDir", "value": job_bundle_path},
         ]
-        config = configparser.ConfigParser()
-
-        set_setting("defaults.farm_id", deadline_resources.farm.id, config)
-        set_setting("defaults.queue_id", deadline_resources.queue_a.id, config)
-
-        job_id: Optional[str] = api.create_job_from_job_bundle(
-            job_bundle_path,
-            job_parameters,
-            priority=99,
-            config=config,
-            queue_parameter_definitions=[],
-        )
-        assert job_id is not None
+        try:
+            with open(os.path.join(job_bundle_path, "template.json"), "w+") as template_file:
+                template_file.write(
+                    json.dumps(
+                        {
+                            "specificationVersion": "jobtemplate-2023-09",
+                            "name": "AssetsExample",
+                            "parameterDefinitions": [
+                                {
+                                    "name": "DataDir",
+                                    "type": "PATH",
+                                    "dataFlow": "INOUT",
+                                    "userInterface": {
+                                        "label": "Input/Output Directory",
+                                        "control": "CHOOSE_DIRECTORY",
+                                    },
+                                },
+                                {"name": "StringToAppend", "type": "STRING"},
+                            ],
+                            "steps": [
+                                {
+                                    "name": "AppendString",
+                                    "script": {
+                                        "actions": {
+                                            "onRun": {"command": "{{ Task.File.runScript }}"}
+                                        },
+                                        "embeddedFiles": [
+                                            {
+                                                "name": "runScript",
+                                                "type": "TEXT",
+                                                "runnable": True,
+                                                "data": append_string_script,
+                                            }
+                                        ],
+                                    },
+                                }
+                            ],
+                        }
+                    )
+                )
+
+            config = configparser.ConfigParser()
+
+            set_setting("defaults.farm_id", deadline_resources.farm.id, config)
+            set_setting("defaults.queue_id", deadline_resources.queue_a.id, config)
```

Comment:
Curious - How is the infra setup done for windows and linux separation, are linux and windows jobs submitted to different queues? AFAIR queue for linux worker has conda environment which doesn't work with queue for windows worker.

Reply:
Synced offline for the test setup, which makes sense.